scout/server/blueprints/panels/controllers.py

```diff
@@ -305,7 +306,26 @@ def new_panel(
     return panel_id
 
 
-def panel_export(store, panel_obj):
+def downloaded_panel_name(panel_obj, format) -> str:
```

Comment:
Also optionally institute and case for the panel extent report! As is the extent report pdf and standard table pdf collide.

Reply:
Ah yes, good suggestion!

Reply:
What did you mean for` panel extent report`? I assumed it was this endpoint: https://github.com/Clinical-Genomics/scout/blob/ef98404273e180d7a6d73ff40d1dccf823a691b6/scout/server/blueprints/panels/views.py#L281 but it already contains institute and case name? 🤔